k8s-eks-addons.tf

```diff
@@ -3,11 +3,13 @@ module "k8s_eks_addons" {
 
   ingress_nginx_config      = merge(var.ingress_nginx_config, { subnets_ids = local.public_subnets })
   cluster_autoscaler_config = var.cluster_autoscaler_config
+  coredns_config            = var.coredns_config
 
   addon_context = {
     aws_caller_identity_account_id = data.aws_caller_identity.current.account_id
     aws_partition_id               = data.aws_partition.current.partition
     aws_region_name                = data.aws_region.current.name
+    eks_cluster_endpoint           = module.eks.eks_cluster_endpoint
```

Comment:
This attribute `eks_cluster_endpoint` is never read.

Reply:
Removed attribute.